Replace oga with REXML

bobmaerten opened the pull request:
Closes #13

bobmaerten committed:
Remove Oga dependency from gemspec

bobmaerten committed:
Rewirte XML parsing code with standard ruby REXML

bobmaerten committed:
Bump gem version to 0.3.0

bobmaerten committed:
Add a real data request test case to be able to run cron jobs on travis

czj commented:
This is a very small gem whose tests run fast and depend on Coliposte's website being up with valid data. We should always test for live values, I think.

czj committed:
Prevent live values tests stubbing

We should always test endpoint availability, and as @bobmaerten coded, stub to test precise values or endpoint unavailability.
